File: C99/1175.c
#include <stdio.h>
#define tamanho 20
int main() {
	int N[tamanho], memoria, i;
	for (i = 0; i < tamanho; i++)
		scanf("%d", &N[i]);
	for (i = 0; i < tamanho / 2; i++) {
		memoria = N[i];
		N[i] = N[tamanho - 1 - i];
		N[tamanho - 1 - i] = memoria;
	}
	for (i = 0; i < tamanho; i++)
		printf("N[%d] = %d\n", i, N[i]);
}

File: Python/1175.py
tamanho = 20
N = [0 for i in range(20)]
memoria = 0
for i in range(tamanho):
    N[i] = int(input())
for i in range(tamanho // 2):
    memoria = N[i]
    N[i] = N[tamanho - i - 1]
    N[tamanho - i - 1] = memoria
for i in range(tamanho):
    print("N[{}] = {}".format(i, N[i]))
